# RNAexpression
Pythonic Approach to analyse RNA Expression

## Setting Up the Analysis Workflow:
Create 1 CSV file for input (User to define path in main.py)
These files serve as inputs to the program and data here should be considered transient 

## Conducting the Analysis 

Copy and paste your results from the BioRad Machine onto your input
A CSV File with calculations will be created 

## Plotting the Relative Gene Expression Results

Run plotting module (Ensure that validation.csv is in the same virtual environment)
Answer the questions as needed
Save as the image that is output

## Input formatting
The program is optimised for the Cq results file from a Biorad machine but should be able to accept other machines with minor adjustments to column names 
An example data has been provided for input to the Analyser module as well as example plotting inputs for the plotting module 

Take note that the example data is not real and are not results of actual scientific experiments.

Target Name should include the Target Gene (What primer targets) name 
Sample Name should include the sample name

## Contributing
Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are greatly appreciated.

Fork the Project Create your Feature Branch (git checkout -b feature/AmazingFeature)

Commit your Changes (git commit -m 'Add some AmazingFeature')

Push to the Branch (git push origin feature/AmazingFeature)

Open a Pull Request
